Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.h

```diff
@@ -79,6 +83,11 @@ WK_CLASS_AVAILABLE(macos(10.11), ios(9.0))
 /*! @abstract Returns the cookie store representing HTTP cookies in this website data store. */
 @property (nonatomic, readonly) WKHTTPCookieStore *httpCookieStore WK_API_AVAILABLE(macos(10.13), ios(11.0));
 
+#if __has_include(<Network/proxy_config.h>)
+// FIXME: Fill in HeaderDocs
+- (void)setProxyConfig:(nw_proxy_config_t)proxyConfig WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
Given this is API, we may want to avoid the abbreviation? `setProxyConfiguration`?
I know we usually frown upon abbreviations in WebKit. Also, there are plenty of APIs with Configuration in their name already (not sure I know one with "Config" in the name.